test/chat_8_3_0_0_model_test.py

```diff
@@ -117,6 +148,14 @@ def test_invite_user_who_then_removes_inviter(self, state):
         state.invite_to_room(invitee=u2, inviter=u1, room_channel=room)
         state.remove_from_room(removee=u1, remover=u2, room_channel=room)
 
+    def test_invite_as_non_owner(self, state):
+        u1 = state.key_alias()
+        u2 = state.key_alias()
+        u3 = state.key_alias()
+        room = state.create_room(creator = u1, room_name='rm1')
+        state.invite_to_room(inviter=u1, invitee=u2, room_channel=room)
+        state.invite_to_room(inviter=u2, invitee=u3, room_channel=room)
```

Comment:
in all these tests, I think we should have an assertion that confirms the operation succeeded or failed

Reply:
The state parameter in those functions is a ChatValidator in `chat_8_3_0_0_state_machine.py`, which has an assertions at the bottom of each function 😄 

https://github.com/symbiont-io/sympl_chat/blob/feature/owner-permissioning/test/chat_8_3_0_0_state_machine.py#L111
```python
    @rule(room_channel=room_channels, inviter=key_aliases, invitee=key_aliases)
    def invite_to_room(self, inviter, room_channel, invitee):
        assume(room_channel != FATAL_ERROR)
        return self.assert_results_match('invite_to_room', inviter, room_channel=room_channel, new_member=invitee)
```
(this was how I inherited it)